lib/typinggame_server/entities/players_rooms.rb

```diff
@@ -0,0 +1 @@
+class PlayersRooms < Hanami::Entity; end
```

Comment:
I believe we don't want to pluralize our entities.

Reply:
@jgayfer updated.